content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -34,7 +34,7 @@ Kubernetes Version | Priority and Preemption State | Enabled by default
 1.9                | alpha                         | no
 1.10               | alpha                         | no
 1.11               | beta                          | yes
-1.14               | GA                            | yes
+1.14               | stable                            | yes
```

Comment:
```suggestion
1.14               | stable                        | yes
```

BTW: does this mean it's more preferable to use the word `stable` over `GA`?

Reply:
I think so. Looks like we use stable instead of 'GA' in our documents.